app/directives/reporting-display/ammap3.js

```diff
@@ -224,11 +218,27 @@ define(['text!./ammap3.html', 'app', 'lodash', 'ammap3', 'ammap3WorldHigh', 'amm
           });
         } //$scope.legendHide
 
+        function averageAssessmentProgress(docs){
+          if(docs.length == 1)
+            return progressToNumber(docs[0].progress_EN_t);
+
+          var progressCount = { 0:0,1:0,2:0,3:0,4:0,5:0 }
+          _.each(docs, function(d){
+            var num = progressToNumber(d.progress_EN_t);
+            progressCount[num]++;
+          });
+          var count = _.reduce(progressCount, function(count, n, k){
+                        return count += (n * parseInt(k));
+                      }, 0);
+
+          return Math.round(count/docs.length);
```

Comment:
is ti possible to have 0 docs? if yes will cause dicsion by 0

Reply:
there will always be 1 or more